tasks.rb

```diff
@@ -2651,7 +2652,12 @@ def set_hpxml_water_heating_systems(hpxml_file, hpxml)
   elsif ['base-dhw-uef.xml'].include? hpxml_file
     hpxml.water_heating_systems[0].energy_factor = nil
     hpxml.water_heating_systems[0].uniform_energy_factor = 0.93
-  elsif ['base-dhw-desuperheater.xml'].include? hpxml_file
+  elsif ['base-dhw-desuperheater.xml',
+         'base-dhw-desuperheater-2-speed.xml',
+         'base-dhw-desuperheater-var-speed.xml'].include? hpxml_file
+    hpxml.water_heating_systems[0].uses_desuperheater = true
+    hpxml.water_heating_systems[0].related_hvac_idref = 'CoolingSystem'
+  elsif ['base-dhw-desuperheater-hpwh.xml'].include? hpxml_file
```

Comment:
This can be combined with above too, right?

Reply:
I started combining due to the new one but finally left out the new one, haha. 🤦